File: tests/cs-sdk/unit/nogui_signal_sender_test.cs
// unit test: SignalSender (nogui)
using System;
using System.Threading.Tasks;
using KnotLink;

var sender = new SignalSender("app.knotlinksdk.test", "test_signal");
await sender.EmitAsync("Hello from Sender!");
Console.WriteLine("Signal sent.");
sender.Dispose();

File: knotlink-sdk-cpp/tests/unit/nogui_signal_sender_test.cpp
// unit test: SignalSender (nogui)
#include "../../knotlink/SignalSender.hpp"
#include <iostream>
using namespace knotlink;

int main() {
    SignalSender sender;
    sender.setConfig("app.knotlinksdk.test", "test_signal");
    sender.emitt("Hello from Sender!");
    std::cout << "Signal sent." << std::endl;
    return 0;
}
